showcase/app/templates/page-components/advanced-table.hbs

```diff
@@ -1318,13 +1515,24 @@
                 mock-state-value={{state}}
                 mock-state-selector="button"
                 @tooltip="Here is more information"
+                @hasResizableColumns={{false}}
```

Comment:
Also opened a ticket to fix this, currently `@hasResizableColumns` is not listed as a bound arg so it is required: https://hashicorp.atlassian.net/browse/HDS-5166